docker/requirements.txt

```diff
@@ -108,6 +108,7 @@ plotly
 ,regex\
 ,ruamel.yaml.clib\
 ,yarl\
+,msgpack\
```

Comment:
```suggestion
```

shouldn't include msgpack as that's managed by a separate PR

Reply:
Sorry, that other PR isn't going to get merged. It's a bit messy, but I want this change in.